### PR TITLE
Change batch_input generation

### PR DESCRIPTION
**Pull request by gengelageTT**
### Ticket
https://github.com/tenstorrent/tt-torch/issues/920

### Problem description
An alternative way to increase batch_size was suggested.

### What's changed
5 Models added in https://github.com/tenstorrent/tt-forge-models/pull/12 are updated to use the follow batch increase method (autoencoder_linear didn't require changes). 

        self.test_input = "This is a sample text from "
        inputs = self.tokenizer.encode_plus(
            self.test_input,
            return_tensors="pt",
            max_length=32,
            padding="max_length",
            truncation=True,
        )
        inputs["input_ids"] = inputs["input_ids"].repeat_interleave(batch, dim=0)
        inputs["attention_mask"] = inputs["attention_mask"].repeat_interleave(batch, dim=0)
        
This requires no changes to tt-torch tests.

### Checklist
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by gengelageTT** [comment]
tt-torch tests pass locally, are being run in CI here: https://github.com/tenstorrent/tt-torch/actions/runs/15783530912

**Review by kmabeeTT — APPROVED** [review]
